src/components/Settings/Settings.tsx

```diff
@@ -1,11 +1,22 @@
-import React from "react";
+import React, { useState } from "react";
 import { useNavigate } from "react-router-dom";
 import { XMarkIcon } from "@heroicons/react/24/solid";
 import { Input } from "@material-tailwind/react";
 
 function Settings() {
   const navigateTo = useNavigate();
 
+  const [deviceID, setDeviceID] = useState("Default Device ID");
```

Comment:
I would have these initialized as empty strings for the time being

Reply:
I left it since it'll eventually initialize to a string that we get from the device - I wanted to show that it was disabled, and that you can't change the box